eng/packages/http-client-csharp-mgmt/generator/Azure.Generator.Management/src/Providers/ResourceClientProvider.cs

```diff
@@ -86,6 +86,8 @@ private ResourceClientProvider(string resourceName, InputModelType model, Resour
 
         protected override string BuildName() => ResourceName.EndsWith("Resource") ? ResourceName : $"{ResourceName}Resource";
 
+        protected override FormattableString BuildDescription() => $"A Class representing a {ResourceName} along with the instance operations that can be performed on it. If you have a ResourceIdentifier you can construct a {Name} from an instance of ArmClient using the GetResource method. Otherwise you can get one from its parent resource <see cref=\"{GetParentResourceTypeName()}\"/> using the Get{ResourceName} method.";
```

Comment:
some type in this description should be a cref.
in order to have a cref, use the `:C` format specifier when the value is a `System.Type` or `CSharpType` like this:
```
=> $"A Class ... If you have a {typeof(ResourceIdentifier):C} ... "
```

Reply:
also we do not have a way of getting parent resource from a resource yet.

Could we first leave the parent resource part a TODO in this formattable string and add a comment with TODO above this?